src/workloads/execution/ColumnStoreIndex.yml

```diff
@@ -312,20 +312,20 @@ BrowserUsageByDistinctUserQuery: &BrowserUsageByDistinctUserQuery
       {"$match": {"metadata.browser": {"$exists": true}}},
       {
         "$addFields": {
-          "metadata.browser_name": {
+          "firstWordOfBrowser": {
```

Comment:
I think that the first word of split "browser" string is the browser name and I would name the field as "browserName" instead of "firstWordOfBrowser".

Reply:
Oh ok! Yeah that's more concise and clear. Done.